utilities/Microsoft.Quantum.Katas/Microsoft.Quantum.Katas.csproj

```diff
@@ -18,7 +18,8 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Jupyter.Core" Version="1.3.60623" />
-    <PackageReference Include="Microsoft.Quantum.IQSharp.Core" Version="0.11.2004.2825" />
+    <PackageReference Include="Microsoft.Quantum.IQSharp.Core" Version="0.11.2005.625-beta" />
```

Comment:
Minor note: For simplicity, you could remove `Microsoft.Jupyter.Core` and `Microsoft.Quantum.IQSharp.Core` from the explicit references list here. The correct versions of those packages will be implicitly pulled in by `Microsoft.Quantum.IQSharp.Jupyter`.

Reply:
Thanks